GUI/src/javafxgui/JavaFXComponents/JFXGameOptionsPage.java

```diff
@@ -0,0 +1,9 @@
+package javafxgui.JavaFXComponents;
+
+import javafx.scene.Parent;
+import javafxgui.GUIView;
+
+public class JFXGameOptionsPage extends Parent {
+    public JFXGameOptionsPage(GUIView guiView) {
+    }
+}
```

Comment:
This is only included here I guess because of the Package changes I made. Sorry, this might be confusing to be here because it has nothing to do with the PR.

I'm not sure on how to exclude the package changes from being added to the commit. Will endeavour to find out.